apps/public/.env.example

```
@@ -5,3 +5,7 @@ DATABASE_PASSWORD=your-password
 DATABASE_NAME=mpga-data
 
 REVALIDATE_SECRET=your-secret-here
+
+MATCH_PLAY_DEADLINE="2026-03-15"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)